content/changelog/2024-01-12-console.md

```diff
@@ -0,0 +1,23 @@
+---
+description: New docs, replicating data with Airbyte, fixes & improvements
+---
+
+### What's new in docs?
+
+- Check out our new guide that describes how to [replicate data from Neon with Fivetran](https://neon.tech/docs/guides/logical-replication-fivetran).
+- Want to better understand how Neon's project billing works? We've added a real-world example of a generative AI project with 80,000 daily active users that walks you through estimating a monthly bill. See [Sample billing](/docs/introduction/billing-sample).
+- Find a typo in the Neon docs? Is there a section that's hard to understand? Notice that a certain topic is missing? The Neon docs are open-source, and contributions are welcome. See our [Documentation Contribution Guide](/docs/community/contribution-guide) for details.
+
+### From the Neon blog
+
+- Learn how to synchronize your PostgreSQL and Airbyte data in this community-contributed blog post by Jacob Prall, Senior Developer Advocate at [Airbyte](https://airbyte.com/): 
+    
+    [A Guide to Logical Replication and CDC in PostgreSQL with Airbyte](https://neon.tech/blog/a-guide-to-logical-replication-and-cdc-in-postgresql-with-airbyte)
+
+### Fixes & improvements
+
+- The [Neon CLI](https://neon.tech/docs/reference/neon-cli) now supports [get](https://neon.tech/docs/reference/cli-branches#get) operations for branches with numeric names. Previously, only string values were supported.
```

Comment:
This was rather a bug than an improvement, but I like this approach nevertheless :D